examples/Example13/example13.cpp

```diff
@@ -196,30 +199,56 @@ int main(int argc, char *argv[])
   OPTION_INT(batch, -1);
   OPTION_BOOL(rotatingParticleGun, false);
 
+  int opt;
+  std::string benchmark_file;
+
+  while ((opt = getopt(argc, argv, "b:")) != -1) {
+    switch (opt) {
+      //-b [benchmark output filename]
+      case 'b':
+        benchmark_file = optarg;
+        break;
+      default:
+        ;
+    }
+  }
+
+  std::string path(argv[0]);
+  if(benchmark_file.empty()) benchmark_file = path.substr(path.find_last_of("/\\") + 1);
+
   vecgeom::Stopwatch timer;
   timer.Start();
 
   NVTXTracer tracer("InitG4");
+  BenchmarkManager<std::string> benchmark_manager;
+  benchmark_manager.timerStart("Total time");
+  benchmark_manager.timerStart("InitG4");
```

Comment:
This can become:
```
BENCHMARK_INIT(); // macro creating the benchmarker if BENCHMARK is set
BENCHMARK_START("Total time");
BENCHMARK_START("InitG4");